packages/terra-menu/src/MenuItem.jsx

```diff
@@ -275,6 +296,31 @@ class MenuItem extends React.Component {
     ]);
 
     let content = textContainer;
+
+    const screenReaderResponse = (
+      <>
+        { MenuUtils.isMac() && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.index' }, { index: index + 1, totalItems })} /> }
+        { MenuUtils.isMac() && (isGroupItem || toggleable) && <VisuallyHiddenText text={markAsToggled ? intl.formatMessage({ id: 'Terra.menu.selected' }) : intl.formatMessage({ id: 'Terra.menu.unselected' })} /> }
+        {/* Adds context for item with submenu-items */}
+        { subMenuItems.length > 0 && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.itemWithSubmenu' })} /> }
+        {/* Adds context for navigating back to parent menu from submenu */}
+        { this.submenu === true && <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.menu.exitSubmenu' })} /> }
+      </>
+    );
+
+    // Adds context for navigating back to parent menu from submenu
+    if (this.itemNode) {
+      this.submenu = this.itemNode.parentNode.getAttribute('data-submenu') === 'true';
+      if (this.submenu) {
+        content = (
+          <>
+            {textContainer}
+            {screenReaderResponse}
+          </>
+        );
+      }
+    }
```

Comment:
why do we need to update content with textContainer and screenReaderResponse here ??

Reply:
Removed above lines in https://github.com/cerner/terra-framework/pull/1626/commits/acb9f635337f19b1d3d0a592313b12f3f3642b1f and updated content in https://github.com/cerner/terra-framework/pull/1626/commits/67f126db5b1c2c1fd2777cfa1a2399da1a979d02 to allow non-toggleable menu items to have custom text embedded